linkerd/protocol/http/src/main/scala/com/twitter/finagle/buoyant/linkerd/LinkerdHeaders.scala

```diff
@@ -286,32 +290,47 @@ object Headers {
           service(req)
         }
       }
+
     }
 
     object Trace {
       val Key = Prefix + "trace"
+      val OpentracingSpanHeader = "x-b3-spanid";
```

Comment:
this has nothing to do with opentracing by the way. OpenTracing includes no library support for B3 unless something changed recently. [census does](https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/HTTP.md#propagation) however. B3 is defined here https://github.com/openzipkin/b3-propagation